judges/assignee-was-attached/assignee-was-attached.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
+# SPDX-License-Identifier: MIT
+
+# Judge that monitors open issue and create missing issue-was-assigned facts
+
+require 'fbe/conclude'
+require 'fbe/octo'
+
+Fbe.conclude do
+  quota_aware
+  on "(and
+        (eq where 'github')
+        (eq what 'issue-was-opened')
+        (empty
+          (and
+            (eq where 'github')
+            (eq repository $repository)
+            (eq issue $issue)
+            (eq what 'issue-was-assigned'))))"
+
+  consider do |f|
+    next if Fbe.fb.query(
```

Comment:
@Yegorov no need to check again, the query above already guarantees that you get proper facts